addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3940,6 +3942,14 @@ const SnippetOptionWidget = Widget.extend({
             if (this.$target[0].classList.contains('o_grid_item') && !isMobileView) {
                 return false;
             }
+            // On mobile, items' reordering is independent from desktop inside
+            // a snippet (left or right), not at a higher level (up or down).
+            if (moveLeftOrRight && isMobileView &&
+                    ColumnLayoutMixin._getItemMobileOrder(this.$target[0])) {
```

Comment:
I know it was like that before but what the hell is that `_computeWidgetVisibility` override doing in the main SnippetOptionWidget definition? :open_mouth:  Should not it be in the `SnippetMove` specific one? :thinking: Which would allow here to not call a private mixin method like that but just do `this._getItemMobileOrder`?

If yes, make a dedicated commit about moving this `_computeWidgetVisibility` definition out of here